data/actionsmap/yunohost.yml

```diff
@@ -578,6 +579,22 @@ app:
                     full: --app
                     help: Write domain & path to app settings for further checks
 
+        ### app_registerurl()
+        registerurl:
```

Comment:
Is there a way to mask this kind of command from cli help ? 
This command is reserved to package script no ?

yunohost app --help is dreadfull...